### DISCUSSION TITLE
Welcome to frxjs-Ngx-Timeline Discussions!

### DISCUSSION THREAD

**Discussion by emanuelefricano93** [emanuelefricano93/frxjs-Ngx-Timeline#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by awmasakityan** [comment]
I've observed that the design undergoes a significant change when the browser's inner width (or viewport width) falls below 900 pixels. Is there a way to prevent this responsive behavior and maintain the original design layout, regardless of the screen size?
